marklogic-data-hub-central/ui/e2e/cypress/integration/explore/crudQueriesfromExplore.spec.tsx

```diff
@@ -356,9 +356,9 @@ describe("save/manage queries scenarios, developer role", () => {
     browsePage.clickColumnTitle(2);
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
-    browsePage.getResetQueryButton().should("be.visible").click();
-    browsePage.getResetConfirmationYes().should("be.visible").click();
-    browsePage.getEditSaveChangesButton().should("be.visible").click();
+    // browsePage.getResetQueryButton().should("be.visible").click();
```

Comment:
@iveyMarklogic Tests are commented out, any reason for that ? Do we have UX approval on this PR

Reply:
@iveyMarklogic The buttons should exist because this PR doesn't deal with any logic of hiding those buttons, @ngodugu-marklogic will take a quick look to confirm if it can be fixed in any another way

Reply:
@iveyMarklogic As discussed, seems like the confirmation pop up is not showing up when we switch queries without saving sorted column. FYI @Sanjeevani19